docs/wiki/framework/arsenal-framework.md

```diff
@@ -402,14 +421,16 @@ All are local.
 1  | Tooltip | String | Optional (default: `""`)
 2  | Picture path | String | Optional (default: `"\z\ace\addons\arsenal\data\iconCustom.paa"`)
 3  | Override a specific button | Number | Optional (default: `-1`)
+4  | Move button on overwrite | Bool | Optional (default: `false`)
 
 Return Value:
 - Successful: Number of the slot (0-9)
 - Error: -1
 
-This function creates a sub category under misc. items in the ACE Arsenal.
-Only items that are listed under 'Misc. Items' are available for sub categories. 
+This function creates a sub-category just above misc items in the ACE Arsenal.
+Only items that are listed under 'Misc. Items' or other sub-categories are available for sub-categories.
 If the 'Override a specific button' argument is not used, the button will added at the bottom of the rest.
+If the 'Move button on overwrite' argument is used, the button will be moved to the bottom of the rest if its position is overriden.
```

Comment:
Looks like one of corrections you made earlier was reverted: `overriden` -> `overridden`